tests/time_filter_test.py

```diff
@@ -1,32 +1,69 @@
-import pytest
 from datetime import datetime
 
-from traveltimepy import PublicTransport, TravelTimeSdk
+import pytest
+
+from traveltimepy.async_client import AsyncClient
+from traveltimepy.requests.common import Property
+from traveltimepy.requests.time_filter import (
+    TimeFilterDepartureSearch,
+    TimeFilterArrivalSearch,
+)
+from traveltimepy.requests.transportation import Driving
 
 
 @pytest.mark.asyncio
-async def test_departures(sdk: TravelTimeSdk, locations):
-    results = await sdk.time_filter_async(
+async def test_departures(async_client: AsyncClient, locations):
+    response = await async_client.time_filter(
         locations=locations,
-        search_ids={
-            "London center": ["Hyde Park", "ZSL London Zoo"],
-            "ZSL London Zoo": ["Hyde Park", "London center"],
-        },
-        transportation=PublicTransport(),
-        departure_time=datetime.now(),
+        departure_searches=[
+            TimeFilterDepartureSearch(
+                id="London center",
+                departure_location_id="London center",
+                arrival_location_ids=["Hyde Park", "ZSL London Zoo"],
+                departure_time=datetime.now(),
+                transportation=Driving(),
+                travel_time=1800,
+                properties=[Property.TRAVEL_TIME],
+            ),
+            TimeFilterDepartureSearch(
+                id="ZSL London Zoo",
+                departure_location_id="ZSL London Zoo",
+                arrival_location_ids=["Hyde Park", "London center"],
+                departure_time=datetime.now(),
+                transportation=Driving(),
+                travel_time=1800,
+                properties=[Property.TRAVEL_TIME],
+            ),
+        ],
+        arrival_searches=[],
     )
-    assert len(results) == 2
+    assert len(response.results) == 2
 
 
 @pytest.mark.asyncio
-async def test_arrivals(sdk: TravelTimeSdk, locations):
-    results = await sdk.time_filter_async(
+async def test_arrivals(async_client: AsyncClient, locations):
+    response = await async_client.time_filter(
         locations=locations,
-        search_ids={
-            "London center": ["Hyde Park", "ZSL London Zoo"],
-            "ZSL London Zoo": ["Hyde Park", "London center"],
-        },
-        transportation=PublicTransport(),
-        departure_time=datetime.now(),
+        arrival_searches=[
```

Comment:
This was a departure search.

Reply:
The method says `test_arrivals` so I am leaving arrival_searches here, it was a mistake in the first place.